exp/addons/internal/webhooks/clusterresourceset_webhook_test.go

```diff
@@ -199,8 +207,9 @@ func TestClusterResourceSetClusterSelectorImmutable(t *testing.T) {
 
 func TestClusterResourceSetSelectorNotEmptyValidation(t *testing.T) {
 	g := NewWithT(t)
-	clusterResourceSet := &ClusterResourceSet{}
-	err := clusterResourceSet.validate(nil)
+	clusterResourceSet := &addonsv1.ClusterResourceSet{}
+	webhook := ClusterResourceSet{}
+	err := webhook.validate(clusterResourceSet, clusterResourceSet)
```

Comment:
```suggestion
	err := webhook.validate(nil, clusterResourceSet)
```
This basically tests the same as ValidateCreate (only a new crs is passed in and no old one)